homeassistant/components/opensky/config_flow.py

```diff
@@ -0,0 +1,85 @@
+"""Config flow for OpenSky integration."""
+from __future__ import annotations
+
+from typing import Any
+
+import voluptuous as vol
+
+from homeassistant.config_entries import ConfigFlow
+from homeassistant.const import (
+    CONF_LATITUDE,
+    CONF_LOCATION,
+    CONF_LONGITUDE,
+    CONF_NAME,
+    CONF_RADIUS,
+)
+from homeassistant.data_entry_flow import FlowResult
+import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers.selector import LocationSelector, LocationSelectorConfig
+from homeassistant.helpers.typing import ConfigType
+
+from .const import DEFAULT_NAME, DEFAULT_RADIUS, DOMAIN
+from .sensor import CONF_ALTITUDE, DEFAULT_ALTITUDE
+
+CONFIG_SCHEMA: vol.Schema = vol.Schema(
```

Comment:
This is only used once, so we can just use it in place without the need for a constant.